src/components/App/App.js

```diff
@@ -24,6 +24,7 @@ const App = () => {
   const [user, setUser] = useState({})
   const [error, setError] = useState("")
   const [galleryPhotos, setGalleryPhotos] = useState([])
+  const [execs, setExecs] = useState([])
 
```

Comment:
Ok so useState gets set as an empty array here so that later on line 66 it can be populated with whatever's returned?

Reply:
Exactly - In order to use that data everywhere in the app, we have to save it in the App component and disperse it as we need!

Reply:
Also useState sets sort of what data type will be in that state later...in this case an array